app/Resources/assets/js/application.js

```diff
@@ -15,6 +15,55 @@ $(function () {
 
     // Activate Bootstrap tooltips.
     $('[data-toggle="tooltip"]').tooltip();
+
+    // For pages with forms, check if there are any "dirty"
+    // inputs and use that to notify the user before navigating away
+    if ( $('form').length > 0 ) {
```

Comment:
phpcs isn't catching this, but we're not using the extraneous spacing within `if`s or arguments of a function, like here and on lines 39, 40, and 60. I will look into fixing the linter rules